pasta_fp.c

```diff
@@ -2152,6 +2152,12 @@ void fiat_pasta_fp_sqrt(uint64_t x[4], const uint64_t value[4]) {
       return;
     }
 
+    uint64_t check[4];
+    fiat_pasta_fp_pow(check, value, P_MINUS_1_OVER_2, P_MINUS_1_OVER_2_LEN);
```

Comment:
should we have it write an error value in the result?

Reply:
Looks like it's used only in once place!  Definitely or just return success or failure.